stdlib/2and3/imaplib.pyi

```diff
@@ -19,8 +25,8 @@ class IMAP4:
     debug: int = ...
     state: str = ...
     literal: Optional[Text] = ...
-    tagged_commands: Dict[str, str] = ...
-    untagged_responses: Dict[str, str] = ...
+    tagged_commands: Dict[bytes, Optional[bytes]]
```

Comment:
The value type seems to be some tuple, because there is a line `self.tagged_commands[tag] = (typ, [dat])`.

Reply:
Good catch.